output_template/python/README.md

```diff
@@ -87,15 +87,15 @@ $ python usb_camera_demo.py -c ../models/meta.yaml -m minimal_graph_with_shape.p
 
 ## Prerequisites
 ```
-Python = 3.5
-Opencv = 3.1.0
-python opencv = 3.1.0
+Python = 3.5, 3.6
+Opencv => 3.1.0
```

Comment:
I've never seen `=>`.
Do you mean `>=`?

Reply:
OH, thanks.

Reply:
@lm-jira Fixed 🙇